src/LDPCDecoders.jl

```diff
@@ -9,27 +9,19 @@ using Statistics
 using RowEchelon
 
 
-# Write your package code here.
-export parity_to_generator
+export
+    parity_check_matrix, decode!,
+    BeliefPropagationDecoder, batchdecode!
```

Comment:
```suggestion
    decode!, batchdecode!, BeliefPropagationDecoder
```

I would suggest not even exporting `parity_check_matrix` for now.